contrib/qpl-cmake/CMakeLists.txt

```diff
@@ -315,7 +315,13 @@ target_compile_definitions(_qpl
         PRIVATE -DQPL_BADARG_CHECK
         PUBLIC -DENABLE_QPL_COMPRESSION)
 
+find_library(LIBACCEL accel-config)
```

Comment:
So, QPL codec in ClickHouse broke with #38245 as it previously relied on dl_open()ing accel-config, right? And it was simply assumed that accel-config is available, correct?

We generally try to build libraries by ourselves for maximum reproducibility. However, I think we can make an exception here since 1. the situation doesn't deteriorate 2. accel-config requires kernel headers [for its build](https://github.com/intel/idxd-config) (which will be time-consuming to pull in, e.g. CI builds fetch all dependencies first) and 3. the QPL codec is currently marked "experimental" anyways.

Reply:
And just for my understanding: Is it right that dl_open() has been removed from recent QPL? (because dl_open() is for sure not happening in the QPL code in ClickHouse's QPL codec code). Also, see my mail from Oct 21.

Reply:
> QPL codec in ClickHouse broke with https://github.com/ClickHouse/ClickHouse/issues/38245 as it previously relied on dl_open()ing accel-config, right? And it was simply assumed that accel-config is available, correct? 

> Is it right that dl_open() has been removed from recent QPL library?

Yes for all questions above.

Reply:
> We generally try to build libraries by ourselves for maximum reproducibility. However, I think we can make an exception here since 1. the situation doesn't deteriorate 2. accel-config requires kernel headers [for its build](https://github.com/intel/idxd-config) (which will be time-consuming to pull in, e.g. CI builds fetch all dependencies first) and 3. the QPL codec is currently marked "experimental" anyways.
Thank you so much for detailed explanation. Meanwhile, we are trying to push forward QPL next release which can be implemented by bundling libaccel-config as static lib. At that time, we won't need "find_library(LIBACCEL accel-config)" any more. I will upgrade QPL codec again by then.